scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -276,16 +276,33 @@
 </a>
 <!-- Submenu content -->
 {% if case.rerun_requested %}
+  {% if config.SHOW_MELTER %}
+    <div id='submenu3' class="collapse">
+      {{ melter_interact(institute, case) }}
```

Comment:
Also here, perhaps rename to something that can be understood and used also by other centers?

Reply:
He, sure, it's just a macro name, so not very exposed. But it doesn't matter, so why not!